Sources/BraintreePayPal/BTPayPalEditRequest.swift

```diff
@@ -7,7 +7,7 @@ import BraintreeCore
 /// Options for the PayPal Edit FI flow
 /// - Warning: This feature is currently in beta and may change or be removed in future releases.
 public class BTPayPalEditRequest {
-    private let token: String
+    let token: String
```

Comment:
I didn't know how to test this with this property marked private.

Reply:
We might not need access to the property directly. I imagine we'll add a `parameters()` method that formats this into JSON and assert on that instead